### ISSUE TITLE
Add it mplug script directory

### ISSUE THREAD

**Issue by z8512** [tomasklaen/uosc#79]
Mplug is like a package manager but for mpv scripts.
Please add your script there (if already not there) and also installation instructions in manifest - https://github.com/Nudin/mpv-script-directory/blob/master/HOWTO_ADD_INSTALL_INSTRUCTIONS.md

**Comment by Zabooby** [comment]
uosc is already [there](https://nudin.github.io/mpv-script-directory/) (crl +f and type uosc).

**Comment by tomasklaen** [comment]
It is there, but old version which is installing from master, not releases, so it may lead to people installing broken stuff.

I've looked into it yesterday in hopes of maybe updating it with correct config, but their API doesn't support installing multiple files from multiple urls, so I couldn't figure out how to tell it to install default `uosc.conf` along with `uosc.lua`, and then I lost interest and moved on to other things 🤐.

I guess it could work if I added the files into a zip, put that into releases, and used their zip installer to extract and place files from there, but I don't like how that adds more things for me to do every release 😞.

Maybe after I get around and restructure the project into multiple files and set up a bundler. Then it wouldn't be that big of a deal to tell it to also zip it for mplug.

And also I don't recall reading if mplug checks versions, which might be an issue, because major upgrades will break stuff for people.
